scripts/build_libpq.sh

```diff
@@ -6,20 +6,27 @@ set -euo pipefail
 set -x
 
 OPENSSL_VERSION="1.1.1d"
-LDAP_VERSION="2.4.48"
+LDAP_VERSION="2.4.50"
 SASL_VERSION="2.1.27"
 # If you change this, fix WANT_LIBPQ too in .travis.yml
 POSTGRES_VERSION="11.5"
+# Fetch architecture
+UNAME_M=`uname -m`
```

Comment:
In the other script you have `TRAVIS_CPU_ARCH` (which I understand is set by travis) and you check for `arm64`. Is it possible to use something similar, maybe have a `CPU_ARCH` variable, and would it be possible to extract `arm64` from the environment somehow, without resorting to the travis-specifc env var?

Reply:
Script which was using `TRAVIS_CPU_ARCH` is set by travis and `lib_pq.sh`  runs in manylinux docker container. So we have to use system commands only to check the architecture. As suggested, modified to `"$(uname -m)" != "aarch64"`. 

Please review and let me know if any more changes required.